src/fee-accounts/components/notifications/fee-account-error-notification.component.html

```diff
@@ -0,0 +1,16 @@
+<ng-container *ngIf="errorMessages.length > 0">
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title">
+        There is a problem
+      </h2>
+      <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <ng-container *ngFor="let errorMessage of errorMessages">
```

Comment:
you may end up with an empty `<ul>` if no error messages (however not sure if this will mess up styles etc if this is moved up)

Reply:
we do have an ngif which looks at the condition errorMessages.length > 0. So, I believe this code would never be executed if no error messages exist. Hope that answers your question